app/views/shared/navigation/_dfe_header_search.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="dfe-header__search">
+  <button class="dfe-header__search-toggle" id="toggle-search" aria-controls="search" aria-label="Open search">
+    <svg class="dfe-icon dfe-icon__search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false" width="27" height="27">
+      <path d="M19.71 18.29l-4.11-4.1a7 7 0 1 0-1.41 1.41l4.1 4.11a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 10a5 5 0 1 1 5 5 5 5 0 0 1-5-5z"></path>
+    </svg>
+    <span class="govuk-visually-hidden"><%= t("search.search") %></span>
+  </button>
+  <div class="dfe-header__search-wrap" id="wrap-search">
+    <form class="dfe-header__search-form" id="search" action="/search" method="get" role="search">
```

Comment:
Do you think we should use a form builder here so that we get the CSRF protection and stuff like that?

Reply:
Added - although I had to leave the SVG button stuff as-is because I have no idea how to do those with form helpers 😆

Reply:
That IS how you do those! The form doesn't add what I thought - but at least we are using it! 👍